test/constexpr_core.cpp

```diff
@@ -179,8 +179,8 @@ RANGES_CXX14_CONSTEXPR auto test_non_member_f(Sequence1234&& a) -> bool {
 RANGES_CXX14_CONSTEXPR auto test_array() -> bool {
     test::array<int, 4> a{{1, 2, 3, 4}};
 
-    auto beg = ranges::begin(a);
-    auto three = ranges::next(beg, 2);
+    // auto beg = ranges::begin(a);
+    // auto three = ranges::next(beg, 2);
```

Comment:
I just tested it and is possible, so i'm fine with commenting out these lines for now. 
I'll switch to safe_int in the constexpr algorithm PR and uncoment them without triggering the warning.